index.html

```diff
@@ -0,0 +1,571 @@
+
+  1. rely on a hardcoded interpretation for directives with certain signatures, or
+  2. accept additional configuration about how to interpret directives in the schema.
+
+The first solution is fragile, particularly as GraphQL has no built-in namespacing mechanisms, so the possibility of name collisions always looms.
+
+The second is unfortunate: GraphQL schemas are generally intended to be self-describing, and requiring additional configuration subtly undermines this guarantee: given just a schema, programs do not necessarily know how to interpret it, and certainly not how to serve it. It also creates the possibility for the schema and configuration to fall out of sync, leading to issues which can manifest late in a deployment pipeline.
+
+## Core Schemas
+
+Introducing **core schemas**.
+
+<div class=hbox>
+  <a class=core>
+    <div class=ring></div>
+    <div class=nucleus>core schema</div>
+  </a>
+</div>
+
+A basic core schema:
+
+<view-of src=basic.graphql title='Basic core schema'></view-of>
+
+**Core schemas** provide a clear and concise mechanism for schema documents to specify the metadata which they provide. Metadata is grouped into **features**, which typically define directives and associated types (e.g. scalars and inputs which serve as directive inputs). Additionally, core schemas provide:
+  - [**Flexible namespacing rules.**](#prefixing) It is always possible to represent any GraphQL schema within a core schema document. Additionally, documents can [choose the names](#as-string) they use for the features they reference, guaranteeing that namespace collisions can always be resolved.
+  - [**Versioning.**](#versioning) Feature specifications follow [semver-like semantic versioning principles](#versioning), which helps schema processors determine if they are able to correctly interpret a document's metadata.
+  - [**Standard rules for extensibility**](#extensibility) of directives and their associated input types. These rules allow specs to annotate each others' metadata, effectively providing "directives on directives," which are otherwise impossible within GraphQL schemas
+
+**Core schemas are not a new language.** All core schema documents are valid GraphQL schema documents. However, this specification introduces new requirements, so not all valid GraphQL schemas are valid core schemas.
+
+The broad intention behind core schemas is to provide a *single document* which provides all the necessary configuration of a *data core*—some program which serves the schema to GraphQL clients, primarily by following directives in order to determine how to resolve queries made against that schema.
+
+### Parts of a Core Schema
+
+When talking about a core schema, we can broadly break it into two pieces:
+- an **export schema** (or **API**) consisting of exported schema elements (objects, interfaces, enums, directives, etc.) which **should** be served to clients, and
```

Comment:
I think it would be helpful to be more explicit somewhere in this section as to what "served to clients" means. I think it basically means both "used to validate and execute GraphQL operations" and "served via introspection". I think being clear that it means both (and especially the former) would be good.

Reply:
Added some language around this.